### ISSUE TITLE
rubocop:disable in HAML file doesn't work

### ISSUE THREAD

**Issue by lucascaton** [brigade/haml-lint#147]
Hi there.

I think I've found a bug: when I try to ignore a cop by using `rubocop:disable` like in the following code:

```haml
-# rubocop:disable Rails/OutputSafety
= @notification.message[:body].html_safe
-# rubocop:enable Rails/OutputSafety
```

It doesn't work, it still complains about the `haml_safe`:

```shell
$ bundle exec haml-lint app/views/notifications/show.html.haml
```

```shell
app/views/notifications/show.html.haml:19 [W] RuboCop: Rails/OutputSafety: Tagging a string as html safe may be a security risk, prefer `safe_join` or other Rails tag helpers instead
```

Cheers.

---

* haml_lint `0.17.1`
* rubocop `0.41.2`
* haml `4.0.7`
* Rails `4.2.6`
* Ruby `2.3.1`

**Comment by sds** [comment]
Thanks for the report, @lucascaton!

Fixed in e03b9cdef497d8a03137babf6c801f3d48899338.

**Comment by lucascaton** [comment]
Thanks @sds!
